src/Simplex/Chat/Messages.hs

```diff
@@ -548,6 +550,21 @@ ciFileEnded = \case
   CIFSRcvError -> True
   CIFSInvalid {} -> True
 
+ciFileLoaded :: CIFileStatus d -> Bool
+ciFileLoaded = \case
+  CIFSSndStored -> True
+  CIFSSndTransfer {} -> True
+  CIFSSndComplete -> True
```

Comment:
There may be no file, and for other statuses it may be removed while copied or fail to be removed - there is a race condition created here. We need to differentiate by file type here.